addon/components/tether-tooltip-and-popover.js

```diff
@@ -27,6 +27,16 @@ function cleanNumber(stringOrNumber) {
 
 export default EmberTetherComponent.extend({
 
+  passedPropertiesObject: null,
+  setPropertiesWithPassedPropertiesObject: on('didReceiveAttrs', function() {
+    this._super(...arguments);
+
+    let passedPropertiesObject = this.get('passedPropertiesObject');
+    if (passedPropertiesObject) {
+      this.setProperties(passedPropertiesObject);
```

Comment:
Ok, so this seems like it's not an issue. @andrew-zenefits pointed me to https://github.com/sir-dunxalot/ember-tooltips/pull/104/files#diff-8096b1ffac251e79ea18b7393d9f241fR68 which re-computes the `passedPropertiesObject` when any `PASSABLE_OPTIONS` change.